src/components/MoneyRequestHeader.tsx

```diff
@@ -79,16 +79,11 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
     const isScanning = TransactionUtils.hasReceipt(transaction) && TransactionUtils.isReceiptBeingScanned(transaction);
     const isPending = TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction);
 
-    const isRequestModifiable = !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
-    const canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
-    let canDeleteRequest = canModifyRequest;
-
-    if (ReportUtils.isPaidGroupPolicyExpenseReport(moneyRequestReport)) {
-        // If it's a paid policy expense report, only allow deleting the request if it's in draft state or instantly submitted state or the user is the policy admin
-        canDeleteRequest =
-            canDeleteRequest &&
```

Comment:
Confirmed in Slack that the admin should not be able to delete requests, the backend also throws an error if we try to. This check was also never true `PolicyUtils.isPolicyAdmin(policy));` , as `canDeleteRequest` assumes the actor is the owner of the action, so it contradicted `isPolicyAdmin`.

Reply:
That PR is outdated given the new logic of instant submit. The backend will not throw if the policy has it enabled.